### ISSUE TITLE
Need support for node 20

### ISSUE THREAD

**Issue by salilag** [okta/okta-oidc-middleware#73]
### Describe the feature request?

Trying to upgrade my app to node 20 and I get this warning
```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@okta/oidc-middleware@5.1.0',
npm WARN EBADENGINE   required: { node: '^12.19.0 || ^14.15.0 || ^16.13.0 || ^18.14.0' },
npm WARN EBADENGINE   current: { node: 'v20.7.0', npm: '10.1.0' }
npm WARN EBADENGINE }
```

### New or Affected Resource(s)

n/a

**Comment by jaredperreault-okta** [comment]
Thanks for the report, we will looking this.

Internal Ref: OKTA-668902

**Comment by jaredperreault-okta** [comment]
@salilag Node support has been released in `5.2.0`

**Comment by salilag** [comment]
Thanks @jaredperreault-okta
